src/strategies/mushrooms/index.ts

```diff
@@ -156,14 +156,11 @@ export async function strategy(
         return [
           address,
           parseFloat(
-            formatUnits(
               (
                 (masterChefResult[address].userInfo.amount *
                   erc20Result.poolMMBalance) /
-                erc20Result.lpTotalSupply
-              ).toString(),
-              18
-            )
+                (erc20Result.lpTotalSupply * 1e18)
```

Comment:
You should use formatUnits instead of `*1e18`, using `*1e18` may break for large number

Reply:
> 
> 
> You should use formatUnits instead of `*1e18`, using `*1e18` may break for large number

```
formatUnits((masterChefResult[address].userInfo.amount * erc20Result.poolMMBalance) / erc20Result.lpTotalSupply).toString(), 18)
```
seems get error:
`Error: invalid BigNumber string (argument="value", value="3617.4759339251227", code=INVALID_ARGUMENT, version=bignumber/5.0.12)`